test/powershell/engine/ResourceValidation/DotNetEventing.Tests.ps1

```diff
@@ -0,0 +1,10 @@
+. "$psscriptroot/TestRunner.ps1"

```

Comment:
To be consistent with other files, maybe change the file name from `DotNetEventing.Tests.ps1` to `***Resource.Tests.ps1`

Reply:
will change